routers/web/web.go

```diff
@@ -1042,6 +1041,14 @@ func registerRoutes(m *web.Router) {
 	}, ignSignIn, context.UserAssignmentWeb(), context.OrgAssignment())
 	// end "/{username}/-": packages, projects, code
 
+	m.Group("/{username}/{reponame}/-", func() {
+		m.Group("/migrate", func() {
+			m.Get("/status", repo.MigrateStatus)
+		})
+	},
+	ignSignIn, context.RepoAssignment, reqRepoCodeReader, context.RepoRef())
+	// end "/{username}/{reponame}/-": migrate
```

Comment:
```suggestion
	}, ignSignIn, context.RepoAssignment, reqRepoCodeReader)
	// end "/{username}/{reponame}/-": migrate
```

`RepoRef` doesn't seem to do anything here.